lib/adapters/config/playwright.ts

```diff
@@ -0,0 +1,114 @@
+import path from 'node:path';
+import crypto from 'node:crypto';
+import _ from 'lodash';
+
+import type {ConfigAdapter} from './index';
+import type {FullConfig, FullProject} from '@playwright/test/reporter';
+import type {TestAdapter} from '../test';
+
+export type PwtProject = FullProject & {
+    snapshotPathTemplate?: string;
+};
+
+export type PwtConfig = FullConfig & {
+    testDir?: string;
+    snapshotDir?: string;
+    snapshotPathTemplate?: string;
+    projects?: PwtProject[]
+}
+
+export const DEFAULT_BROWSER_ID = 'chromium';
+const DEFAULT_SNAPSHOT_PATH_TEMPLATE = '{snapshotDir}/{testFileDir}/{testFileName}-snapshots/{arg}{-projectName}{-snapshotSuffix}{ext}';
+
+export class PlaywrightConfigAdapter implements ConfigAdapter {
+    private _config: PwtConfig;
+    private _browserIds: string[];
+
+    static create<T extends PlaywrightConfigAdapter>(this: new (config: PwtConfig) => T, config: PwtConfig): T {
+        return new this(config);
+    }
+
+    constructor(config: PwtConfig) {
+        this._config = config;
+        this._browserIds = _.isEmpty(this._config.projects) ? [DEFAULT_BROWSER_ID] : this._config.projects.map(prj => prj.name).filter(Boolean);
```

Comment:
if the user does not specify the browser in the `projects` section then `chromium` will be used by default. So I support this behaviour.